src/components/ProfilePage/AddressBox/AddressBox.tsx

```diff
@@ -14,6 +14,9 @@ import {
   PassImageBox,
   ProfileIconButton,
 } from './AddressBox.style';
+import useImageStatus from 'src/hooks/useImageStatus';
+
+const DEFAULT_IMAGE = '/default_effigy.svg';
```

Comment:
Would be good to refactorize it to avoid duplication

Reply:
+1